src/PKSim.Infrastructure/ProjectConverter/v5_5/Converter551To552.cs

```diff
@@ -174,7 +174,10 @@ private void addSurfaceAreaVariabilityTo(Population population)
          //use a clone as this parameter will be used to create the advanced parameter
          var clone = _cloner.Clone(factor);
          var advancedParameter = new AdvancedParameter {DistributedParameter = clone, Seed = population.Seed};
-         var randomValues = advancedParameter.GenerateRandomValues(population.NumberOfItems);
+         //Because of changes in population serialization in v9, NumberOfItems in population is not set yet. Let's retrieve it from the internal structure
+         var numberOfIndividuals = population.IndividualValuesCache.AllParameterValues.FirstOrDefault()?.Count;
```

Comment:
I have fixed the test for now as it is possible to retrieve the number of individuals from the object structure